packages/strapi-design-system/src/v2/Breadcrumbs/Crumb.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Typography } from '../../Typography';
+
+export const Crumb = ({ children, ...props }) => (
+  <Typography variant="pi" textColor="neutral800" fontWeight="bold" {...props}>
```

Comment:
Crumb shouldn't be always bold
The isLast should be bold regardless of which component it is (Crumb, CrumbLink, CrumbSimpleMenu

Reply:
Done in d6f30e4fcfd97bdf2fb73214247ccb5c6ca53a3f